Missed-Homework-Part_1/Lecture-10/Task-10.2.py

```diff
@@ -0,0 +1,13 @@
+"""
+Დაწერეთ ფუნქცია რომელსაც გადაეცემა განუზღვრელი რაოდენობის რიცხვიტი
+ტიპის არგუმენტი და აბრუნებს მაქსიმალურს. Გამოიძახეთ ფუნქცია რამდენჯერმე
+სხვადასხვა არგუმენტისთვის რომ აჩვენოთ მისი მუშაობა
+"""
+
+def maximum(*args):
+    print(max(*args))
```

Comment:
ზოგადად, ფუნქციას იმისთვის ვქმნით რომ შიგნით ლოგიკა დავწეროთ და ფუნქციაში შიგნით მხოლოდ მზა ფუნქცია თუ გამოვიყენეთ 1 ხაზისთვის ამ ფუნქციის შექმნა არამართებულია, გთავაზობ ალტერნატივას:
def my_max(*args):
_max = args[0]
for i in args:
if type(i) == int or type(i) == float:
if i >_max:
_max = i
return _max

print(my_max(1,2,"car",-7.5,25,15,27))